test/models/user_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
+
+class UserTest < ActiveSupport::TestCase
+  def setup
+    @user = User.new(name:"Example User", email: "userexaample.com")
+  end
+  
+  test "should be valid" do
+    assert @user.valid?
+  end
+  test "name should be present" do
+    @user.name = ""
+    assert_not @user.valid?
+  end
+   test "name should not be too long" do
+    @user.name = "a"*21
+    assert_not @user.valid?
+   end
+end
```

Comment:
This one looks good. Only concern is you added a validation for email uniqueness but it is not tested

Reply:
These are enough validations for this exercise, but for future projects, you could add more validations like testing that the email is a valid string with a regex or making sure that the email is stored as lowercase.